edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -191,9 +200,15 @@ const CardScreenComponent: ReactComponent<{ deckId: string, cardIndex: number; i
                 </TView>
                 <TView style={styles.lastButtonHalf}>
                     <FancyButton
+                        loading={isLoading}
                         testID='gotItButton'
                         backgroundColor='green'
-                        onPress={() => updateLearningStatusCard(deckId, cardIndex, updateCard, "Got it", card)}
+                        onPress={async () => {
+                            setIsLoading(true);
```

Comment:
You may want to create a function for this, just like you did for updateLearningStatusCard.

Reply:
I can do that so it is more readable but at the same time I think that this function is quite short and it is clear what it is doing so in my opinion, we don't need to create an extra function for this. But if there is a more complex, bigger function, it is absolutely a must to create a function outside to increase the readability of the code.